src/core/time.ts

```diff
@@ -202,6 +222,14 @@ export class TimeValue {
     }
   }
 
+  isForever(): boolean {
+    if (this.seconds === Number.MAX_SAFE_INTEGER && this.nanoseconds === 0) {
```

Comment:
I don't think we care about the nanoseconds. If we say that `(Number.MAX_SAFE_INTEGER, 0)` is :infinity: then shouldn't `(Number.MAX_SAFE_INTEGER, 0)` also be :infinity:?
```suggestion
    if (this.seconds === Number.MAX_SAFE_INTEGER) {
```